integration/local-gadget/trace_tcpconnect_test.go

```diff
@@ -31,21 +31,16 @@ func TestTraceTcpconnect(t *testing.T) {
 		Cmd:          fmt.Sprintf("local-gadget trace tcpconnect -o json --runtimes=%s", *containerRuntime),
 		StartAndStop: true,
 		ExpectedOutputFn: func(output string) error {
-			expectedEntries := []*tcpconnectTypes.Event{
-				{
-					Event:     BuildBaseEvent(ns),
-					Comm:      "wget",
-					IPVersion: 4,
-					Daddr:     "1.1.1.1",
-					Dport:     80,
-				},
-				{
-					Event:     BuildBaseEvent(ns),
-					Comm:      "wget",
-					IPVersion: 4,
-					Daddr:     "1.1.1.1",
-					Dport:     443,
-				},
+			saddr := GetTestPodIP(ns, "test-pod")
+			NginxIP := GetTestPodIP(ns, "nginx-pod")
```

Comment:
nit: if you move the `commandsPreTest` before this, then you can reuse the NginxIP from L68 without having to call `GetTestPodIP(ns, "nginx-pod")` again.